ros2topic/ros2topic/verb/bw.py

```diff
@@ -93,12 +96,20 @@ def __init__(self, node, window_size):
         self.sizes = []
         self.times = []
         self.window_size = window_size
+        self.use_sim_time = False
+        if node.has_parameter('use_sim_time'):
+            self.use_sim_time = node.get_parameter('use_sim_time')
+        if self.use_sim_time:
+            self.clock = node.get_clock()
 
     def callback(self, data):
         """Execute ros sub callback."""
         with self.lock:
             try:
-                t = time.monotonic()
+                if self.use_sim_time:
+                    t = self.clock.now().nanoseconds * 1.e-9
+                else:
+                    t = Clock(clock_type=ClockType.STEADY_TIME).now()
```

Comment:
Creating `Clock` object every callback would be not cost effective? maybe we can prepare the object before and call `now()` only in this callback function?

Reply:
Yes, updated.